tests/test_diffeq/test_odefiltsmooth/test_prior.py

```diff
@@ -53,34 +53,68 @@
 
 class TestIBM(unittest.TestCase, NumpyAssertions):
     def setUp(self):
-        self.ibm = prior.IBM(2, 2, DIFFCONST)
+        self.prior = prior.IBM(2, 2, DIFFCONST)
 
     def test_chapmankolmogorov(self):
-        mean, cov = np.ones(self.ibm.ndim), np.eye(self.ibm.ndim)
+        mean, cov = np.ones(self.prior.ndim), np.eye(self.prior.ndim)
         initrv = RandomVariable(distribution=Normal(mean, cov))
-        cke, __ = self.ibm.chapmankolmogorov(0.0, STEP, STEP, initrv)
+        cke, __ = self.prior.chapmankolmogorov(0.0, STEP, STEP, initrv)
         self.assertAllClose(AH_22_IBM @ initrv.mean(), cke.mean(), 1e-14)
         self.assertAllClose(
             AH_22_IBM @ initrv.cov() @ AH_22_IBM.T + QH_22_IBM, cke.cov(), 1e-14
         )
 
+    def test_chapmankolmogorov_super_comparison(self):
+        """
+        The result of chapmankolmogorov() should be identical to the matrix fraction decomposition technique
+        implemented in LinearSDE, just faster.
+        """
+        mean, cov = np.ones(self.prior.ndim), np.eye(self.prior.ndim)
+        initrv = RandomVariable(distribution=Normal(mean, cov))
+        cke_super, __ = super(
+            type(self.prior), self.prior
+        ).chapmankolmogorov(  # pylint: disable=bad-super-call
+            0.0, STEP, STEP, initrv
+        )
```

Comment:
I would have probably put the pylint ignore here, at the end of the full expression. Maybe that fixes it?
Alternatively, just write it into the line right before (74 then), I don't think it should be a problem if the disable is active for the whole funciton, and it might be more readable.